### PR TITLE
Bug fix #1010

### LINKED ISSUE #1010: records-ui: add a simple "contact the author" button with contact form to the owner of the record.
Currently it's not easy to contact the owner of the record because we don't expose user emails.
We can however allow for contacting them through us and send them an email from the users.

It should be possible to opt-out of this (bool in the userprofile model).

### PR DESCRIPTION
**Pull request by vedantrathore**
@krzysztof sorry for the delay, I'll complete this in a day now :sweat_smile: I have added the template for the submission form, I am currently facing some errors on how to write the python code for email part, any suggestions would be highly appreciated.

### CODE REVIEW

**Comment by vedantrathore** [comment]
@krzysztof I've added some code for sending email, but not sure on how to proceed forward, could you please help :)

File: zenodo/modules/records/views.py [hunk 1]
```diff
@@ -53,6 +55,43 @@
...
+
+    """If form is validated send email to the admin."""
+    if form.validate_on_submit():
+        """Dictionary storing data to be sent."""
+        context = dict(
+            form=form,
+            current_user=current_user,
+        )
+
+        # How to implement recipient user email?
```
Comment (vedantrathore) [inline zenodo/modules/records/views.py]: @krzysztof How do I add the recipient's (i.e. the person whose record is being investigated here) email address here? is there a method for getting user email from record PID?

Comment (vedantrathore) [inline zenodo/modules/records/views.py]: @krzysztof How do I add the recipient's (i.e. the person whose record is being investigated here) email address here? is there a method for getting user email from record PID?

**Comment by vedantrathore** [comment]
I've added the code but not tested it yet, since it's under development. I will write the unit test after the feature is done

**Comment by lnielsen** [comment]
Closed via #1008 #1239 and #1243

**Pull request closed without merging**